presto-main/src/main/java/com/facebook/presto/server/ResourceGroupInfo.java

```diff
@@ -51,6 +52,7 @@
     private final List<ResourceGroupInfo> subGroups;
     private final List<QueryStateInfo> runningQueries;
 
+    @JsonCreator
```

Comment:
Why is it needed?

If it is needed indeed - we should explicitly annotate all parameters of the constructor as we do for other Json serializable objects (e.g.: https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java#L82)